backend/apps/notifications/tests/test_ws_leak.py

```
@@ -0,0 +1,168 @@
+from apps.notifications import metrics
+from config.asgi import application
+
+User = get_user_model()
+
+HEADERS = [(b"origin", b"http://localhost")]
+CHURN_CONNECTIONS = 50
+
+
+@pytest.fixture
+def auth_user(db):
+    return User.objects.create_user(username="leakuser", password="testpassword123")
```

Ruff (0.16.0)

[error] Possible hardcoded password assigned to argument: "password"

(S106)